pilot/pkg/config/kube/crdclient/gen/types.go.tmpl

```diff
@@ -22,18 +22,23 @@ package crdclient
 // as declared in the Istio config model.
 
 import (
+	"bytes"
 	"context"
 	"fmt"
+	"encoding/json"
 	metav1alpha1 "istio.io/api/meta/v1alpha1"
 
 	versionedclient "istio.io/client-go/pkg/clientset/versioned"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	yamlv3 "gopkg.in/yaml.v3"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	gatewayapiclient "sigs.k8s.io/gateway-api/pkg/client/clientset/gateway/versioned"
+	"github.com/ghodss/yaml"
```

Comment:
as mentioned, do not need 2 yaml libraries. Use `gopkg.in/yaml.v3` in very rare cases when we need complex things and sigs.k8s.io/yaml for everything else

Reply:
package removed